### ISSUE TITLE
Exception when rendering login button (FBLoginManager.getCredentials)

### ISSUE THREAD

**Issue by madjam002** [magus/react-native-facebook-login#8]
I get the following error when rendering the login button using the code in the README

![Screenshot](http://i.imgur.com/fz4Mwt1l.png)

As far as I'm aware, I've setup the library correctly and added the correct keys to my plist file.
`FBLoginManager.getCredentials` seems to be causing the problem.

**Comment by magus** [comment]
Can you open the example project and let me know if that fails? If it does send me your package.json so I can try to reproduce the issue. If it doesn't then you can paste your code so I can try to debug.

**Comment by madjam002** [comment]
Can't build the example project - I get the error
`/react-native-facebook-login/example/iOS/AppDelegate.m:13:9: 'FBSDKCoreKit/FBSDKCoreKit.h' file not found`

I'm new to iOS development so I'm not entirely sure what I'm doing, but I did `npm install` in the examples folder and the react-native-facebook-login root too.
The example package.json appeared to use an out of date version of react-native-facebook-login, upgrading it didn't help. The example project also links the FBSDK frameworks which I thought would have been the problem, so I'm a bit unsure about this...

**Comment by magus** [comment]
I'm new to iOS development too, so thanks for bearing with me!

I've corrected the framework search paths which were incorrectly set to absolute paths by XCode, which should resolve the example project failing to build. In addition, the example will now use the latest 0.* release. 

Finally, your initial issue was totally due to a bug I introduced just the day before which I've squashed and it should get picked up as I push here in a moment.

Thanks!
